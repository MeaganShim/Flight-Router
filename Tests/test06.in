n 11
i 4;2;46.48
i 6;7;56.93
i 4;3;59.86
i 4;4;5.10
i 7;8;13.28
i 4;0;98.14
i 0;3;25.48
i 5;3;68.84
i 3;7;63.87
i 2;5;86.05
i 10;2;33.78
i 10;9;29.08
i 0;5;52.75
i 9;0;12.89
i 3;0;22.15
i 1;5;7.13
i 0;0;29.34
i 0;7;26.00
i 8;5;63.22
i 7;3;85.73
i 4;9;91.10
i 4;2;34.98
i 3;1;83.58
i 4;3;73.73
i 9;1;1.23
i 8;1;58.93
i 1;1;81.09
i 10;6;57.83
i 1;0;60.94
i 2;10;25.21
i 6;3;89.22
i 10;4;0.06
i 3;4;2.82
i 1;9;48.97
i 0;0;56.71
i 7;1;37.80
i 5;6;42.78
i 2;0;26.40
i 1;8;79.01
i 4;1;63.14
i 1;2;51.59
i 3;7;79.35
i 0;3;8.80
i 1;1;89.26
i 7;10;1.08
i 6;7;51.86
i 9;8;0.84
i 1;1;93.36
i 5;7;41.73
i 7;7;67.64
i 2;3;54.73
adjacent 3;4;2.82
adjacent 7;6;31.57
mst 4
mst 3
mst 8
mst 1
mst 9
mst 10
mst 1
mst 7