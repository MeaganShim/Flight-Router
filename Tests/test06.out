success
success
success
success
illegal argument
success
success
success
success
success
success
success
success
success
success
success
success
illegal argument
success
success
success
success
success
success
success
success
success
illegal argument
success
success
success
success
success
success
success
illegal argument
success
success
success
success
success
success
success
success
illegal argument
success
success
success
illegal argument
success
illegal argument
success
adjacent 3 4 2.82
not adjacent 7 6 31.57
mst 139.41
mst 139.41
mst 139.41
mst 139.41
mst 139.41
mst 139.41
mst 139.41
mst 139.41
